crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -49,6 +49,18 @@ pub trait SystemParam: Sized {
 /// Additionally, it is the implementor's responsibility to ensure there is no
 /// conflicting access across all SystemParams.
 pub unsafe trait SystemParamState: Send + Sync + 'static {
+    /// Values of this type can be used to adjust the behavior of the
+    /// system parameter.  For instance, this can be used to pass
+    /// values from a `Plugin` to a `System`, or to control the
+    /// behavior of the `System`.
+    ///
+    /// The default configuration of the parameter is set by
+    /// [`SystemParamState::default_config`].  To change it, invoke
+    /// [`FunctionSystem::config`] when creating the system.
+    ///
+    /// See [`FunctionSystem::config`] for more information and examples.
+    ///
+    /// [`FunctionSystem::config`]: struct.FunctionSystem.html#method.config
```

Comment:
could this link be replaced by an intra doc link?

Reply:
Apparently it can.  My first attempt didn't produce a working link, but linking to `super::FunctionSystem::config` works.